src/opt-out.html

```diff
@@ -61,13 +61,10 @@ <h3 class="govuk-heading-s" id="mobile-network-tests">
         Mobile phone network tests
       </h3>
       <p class="govuk-body">
-        Mobile network operators in the UK are <a href="/alerts/planned-tests">testing emergency alerts</a>.
-      </p>
-      <p class="govuk-body">
-        A small number of Android phones and tablets may receive these test alerts.
+        If you have an Android device, there’s a small chance you may get <a href="/alerts/planned-tests">test alerts</a> from your mobile network operator.
       </p>
       {% set testAlertButton %}
-        If you receive a mobile network operator test and want to opt out
+        If you receive a test alert and want to opt out
```

Comment:
I think it’s important to keep the word operator in here because the alert will be titled:
> ⚠️ Emergency Alert (Operator)